### PR TITLE
track new stream vaults tvl

### PR DESCRIPTION
**Pull request by 0xDro**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/stream exports TVL: 
        
 
 
    ethereum                  1.57 M
    
    total                    1.57 M

File: projects/stream/index.js [hunk 1]
```diff
@@ -26,15 +29,39 @@ async function tvl(_, _1, _2, { api }) {
...
+    abi: abi.totalBalance,
+    target: HODLWBTC_VAULT_CONTRACT,
+  });
+
+  const weth_totalBalance = await api.call({
+    abi: abi.totalBalance,
+    target: HODLWETH_VAULT_CONTRACT,
+  });
+
+  const usdw4_total = (usdw4_totalSupply - usdw4_vaultBalance) / (vaultToken / baseToken) / 10 ** usdw4_decimals;
```
Comment (0xngmi) [inline projects/stream/index.js:47]: why is `(vaultToken / baseToken)` not applied to the other vaults?

Comment (0xDro) [reply projects/stream/index.js:47]: usdw4 and all other vaults are different contracts. The other 3 track underlying tokens in contract state directly so simply call a getter that returns amountLocked + current contract balance (funds are held off chain)

**Comment by llamatester** [comment]
The adapter at projects/stream exports TVL: 
        
 
 
    ethereum                  1.59 M
    
    total                    1.59 M

**Comment by g1nt0ki** [comment]
@0xDro thanks for the PR

**Pull request merged**